typescript/lib/bulk_command_execution/v0_bulk_command_api/v0_bulk_commands.ts

```diff
@@ -0,0 +1,260 @@
+import { LoadLambdaArgs } from "../../../kurtosis_core_rpc_api_bindings/api_container_service_pb";
+import { newEmptyExecCommandArgs, newEmptyExecuteBulkCommandsArgs, newEmptyExecuteLambdaArgs, newEmptyGenerateFileArgs, newEmptyLoadLambdaArgs, newEmptyLoadStaticFilesArgs, newEmptyRegisterServiceArgs, newEmptyRemoveServiceArgs, newEmptyRepartitionArgs, newEmptyStartServiceArgs, newEmptyWaitForEndpointAvailabilityArgs } from "../../../lib/constructor_calls";
+import { ExecCommandArgs, ExecuteBulkCommandsArgs, ExecuteLambdaArgs, GenerateFilesArgs, LoadStaticFilesArgs, RegisterServiceArgs, RemoveServiceArgs, RepartitionArgs, StartServiceArgs, WaitForEndpointAvailabilityArgs } from "../../../kurtosis_core_rpc_api_bindings/api_container_service_pb";
+import { V0CommandTypeVisitor, V0CommandType, acceptVisitor } from "./v0_command_types";
+import { ok, err, Result } from "neverthrow";
+import * as jspb from "google-protobuf";
+
+// ====================================================================================================
+//                                   Command Arg Deserialization Visitor
+// ====================================================================================================
+
+// Visitor that will be used to deserialize command args into
+class cmdArgDeserializingVisitor implements V0CommandTypeVisitor {
+	private readonly bytesToDeserialize: string; //TODO (comment) - string because no byte type in typescript
+	private deserializedCommandArgsPtr: jspb.Message;
+
+    constructor (bytesToDeserialize: string) {
+        this.bytesToDeserialize = bytesToDeserialize;
+    }
+
+    public visitLoadLambda(): Result<null, Error> {
+        let args: LoadLambdaArgs = newEmptyLoadLambdaArgs();
```

Comment:
Due to the differences in Golang vs JS, you don't actually need to instantiate this to an empty object since it will just be thrown away anyways when `args = JSON.parse(...)` runs.
`let args: LoadLambdaArgs` will be sufficient